supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesConfiguration.scala

```diff
@@ -2,7 +2,19 @@ package cromwell.backend.impl.tes
 
 import cromwell.backend.BackendConfigurationDescriptor
 
+import net.ceedubs.ficus.Ficus._
+
 class TesConfiguration(val configurationDescriptor: BackendConfigurationDescriptor) {
+
   val endpointURL = configurationDescriptor.backendConfig.getString("endpoint")
   val runtimeConfig = configurationDescriptor.backendRuntimeAttributesConfig
+  val useBackendParameters =
```

Comment:
The assumption is that this config option will be removed, and behavior will default to `true`, when full TES 1.1 is released.

Reply:
I wonder if we might end up keeping the parameter but changing the default. That way, it could still be flipped back in case anyone else using the TES backend runs into problems. We could also deprecate the parameter and encourage people to update their systems so we can eventually remove it.

Reply:
This behavior is part of the TES 1.1 spec - we could consider putting full TES 1.0 behavior behind a config flag when we release official TES 1.1 support, but once the spec is released and we support it, I don't really want to give users the ability to turn different parts of it on and off. 

Thinking about a hypothetical Cromwell v100 that releases full TES 1.1 support and removes this config field, what are different users' experiences? Anyone with this option set to True doesn't see a change in this behavior (though they lose the ability to turn it on and off, which I wouldn't predict they were using). Anyone else will just get a single-step upgrade to TES 1.1. I think both those experiences are OK. Plus, this config option is unpublished, so people should only be using it in consultation with us (one would hope??).

Reply:
Yeah, I'm probably being too paranoid here... or been on the receiving end of non-backwards-compatible changes too often to not have empathy for the hypothetical future person who says, "I just need to update to the latest version because of a security patch and now I have to also change my system to keep it working."

Reply:
Always better to be too paranoid than not paranoid enough 🙂 I do think this is something we'll have to think carefully about when we do the full TES 1.1 update, since that will change the default behavior for those security-updaters, and I'm not sure what else will end up included in that version of the spec.